src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2022,28 +2051,13 @@ public override BoundNode VisitConversion(BoundConversion node)
             {
                 switch (node.ConversionKind)
                 {
-                    case ConversionKind.ExplicitUserDefined:
-                    case ConversionKind.ImplicitUserDefined:
```

Comment:
>case ConversionKind.ImplicitUserDefined: [](start = 20, length = 40)

It is not obvious why it is no linger needed to handle these cases.

Reply:
Handled in `ApplyConversion`.

---
In reply to: [183861742](https://github.com/dotnet/roslyn/pull/26356#discussion_r183861742) [](ancestors = 183861742)